src/site/layouts/news_story.drupal.liquid

```diff
@@ -58,7 +58,8 @@ Example data:
 {% include "src/site/includes/header.html" with drupalTags = true %}
 {% include "src/site/includes/alerts.drupal.liquid" %}
 {% include "src/site/includes/preview-edit.drupal.liquid" %}
-{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" with hideHomeBreadcrumb = true %}
```

Comment:
Ah man...I personally dread these kinds of things, because this touches a lot of files, so it isn't easy to revert in case design changes their mind. I'm not opposed to just removed the `Home` breadcrumb in the `componentDidMount` of your SideNav. Then if the design is finalized, we can make the template reflect that change.

Up to you though. It's just a pain to manage this.

Reply:
Oh totally, I would LOVE to do this in `componentDidMount` of our SideNav, but unfortunately this is higher up and just removing it within my SideNav component wouldn't fix the problem since the SideNav doesn't render the breadcrumbs -- so I would need to change it in the GraphQL response somewhere 🤔

Reply:
Yeah, as like a complete short-term hack until design is final, you could do `document.querySelectorAll` to find it, if that works/makes sense, since it seems like page that renders this side-nav should omit the Home breadcrumb

Reply:
Yupp that makes sense! My only concern is that the ask was to remove it only from the above pages on non-prod envs, so if there's other pages that are rendering the "Home" breadcrumb it may not be desired to remove it completely 😬If that's not a valid concern though I am happy to approach it using `document.querySelectorAll`!

Reply:
I hypothesize this is going to be a short-term thing anyways since ideally all breadcrumbs would be managed via Drupal...

Reply:
Doesn't this component only mount on non-prod environments? Sorry, not sure I follow, but sounds like you know what you're doing 🙂